examples/2d_isometry/cars.rs

```diff
@@ -0,0 +1,138 @@
+use amethyst::assets::{AssetStorage, Loader};
+use amethyst::core::{IsometricTransform, Time};
+use amethyst::ecs::{Component, Join, Read, System, VecStorage, World, WriteStorage};
+use amethyst::prelude::*;
+use amethyst::renderer::{
+    FilterMethod, MaterialTextureSet, PngFormat, SamplerInfo, Sprite, SpriteRender, SpriteSheet,
+    SpriteSheetHandle, Texture, TextureCoordinates, TextureMetadata, Transparent, WrapMode,
+};
+
+use map::UNIT_DIMENSIONS;
+
+const CARS_TEXTURE_ID: u64 = 1;
+
+pub struct Car {
+    top: f32,
+    bottom: f32,
+    speed: f32,
+    up_sprite: usize,
+    down_sprite: usize,
+}
+
+impl Component for Car {
+    type Storage = VecStorage<Self>;
+}
+
+pub fn initialise_cars(world: &mut World, sprite_sheet: SpriteSheetHandle) {
+    let mut iso_transf =
+        IsometricTransform::from_unit_dimensions(UNIT_DIMENSIONS.0, UNIT_DIMENSIONS.1);
+    iso_transf.translation.x = 1.5;
+    iso_transf.translation.y = 1.3;
+    iso_transf.order_priority = 300.0;
+    world
+        .create_entity()
+        .with(iso_transf)
+        .with(Car {
+            top: 2.3,
+            bottom: 0.0,
+            speed: 0.5,
+            up_sprite: 0,
+            down_sprite: 1,
+        })
+        .with(SpriteRender {
+            sprite_sheet,
+            sprite_number: 0,
+            flip_horizontal: false,
+            flip_vertical: false,
+        })
+        .with(Transparent)
+        .build();
+}
+
+pub fn load_cars_sprite_sheet(world: &mut World) -> SpriteSheetHandle {
+    let texture_handle = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            "texture/vehicles_blue.png",
+            PngFormat,
+            TextureMetadata::default()
+                .with_sampler(SamplerInfo::new(FilterMethod::Trilinear, WrapMode::Border)),
+            (),
+            &texture_storage,
+        )
+    };
+
+    let mut material_texture_set = world.write_resource::<MaterialTextureSet>();
+    material_texture_set.insert(CARS_TEXTURE_ID, texture_handle);
+
+    let mut sprites = Vec::new();
+    let tex_coords = TextureCoordinates {
+        left: 1.0 / 256.0,
```

Comment:
these are the texture dimensions?

Reply:
Why does it start at `1.0` and the width is different from `right - left` in unnormalized texture coordinates?

Reply:
The spritesheet is blurry which makes it hard to be pixel perfect.
Now regarding why I chose 32px, it's because I assumed it's what would make sense for the sprite size, but again look at the spritesheet, it's weird.